### DISCUSSION TITLE
The New Nike Unicorn Rainbow Air Shoe (VQGAN +CLIP)

### DISCUSSION THREAD

**Discussion by gateway** [rkhamilton/vqgan-clip-generator#26] [Share generated art]
Source Image:
![nike](https://user-images.githubusercontent.com/519563/138497393-2e927eaa-bbb5-4560-8c8f-b0a0d1d72bee.jpg)

Final Image:
![unicorn-nike](https://user-images.githubusercontent.com/519563/138497424-45b50a97-5b79-41eb-83eb-0ec0626c4083.jpg)

Text: "A Nike shoe that turns into a unicorn with rainbows"
Iterations: 200
Save Every: 10

**Comment by rkhamilton** [comment]
Nice! My daughter would be all over that.
